EIPS/eip-XXXX.md

```diff
@@ -0,0 +1,177 @@
+
+## Abstract
+While building the *[Light Client Infrastructure for the Teku Client (for Ethereum 2)](https://github.com/jeyakatsa/teku/tree/master/light-client),* a discovery was made; clients serve as arbiters or "bridges" to the Ethereum main chain.
+| ETH2 Client   | Language     | Team                |
+|:--------------|:------------ |:------------------- |
+| Teku          | Java         | Consensys           |
+| Lighthouse    | Rust         | Sigma Prime         |
+| Trinity       | Python       | Ethereum Foundation |
+| Prysm         | Go           | Prysmatic Labs      |
+
+Such clients offer differing languages that connect with the transactions offered on Ethereum, thus concluding if said clients can offer a base layer capable of inferring transactions from Ethereum's base layer, can tokens created in said languages also offer the same results?
+
+## Motivation
+Today, millions of websites are created every week in many languages. What if the Ethereum ecosystem could also do the same, but with currencies and tokens?
+
+## Specification
+>The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+#### Starting Point:
+Creating a library of implementing smart-contracts in these languages: Java, Rust, Python & Go, beginning with Java. The first agenda being creating a simple smart-contract out of Java by using Solidity as the base layer.
+
+##### Smart-Contract Storage Example in *Solidity*:
+```solidity
+pragma solidity >=0.4.16 <0.9.0;
+
+contract SimpleStorage {
+    uint storedData;
+
+    function set(uint x) public {
+        storedData = x;
+    }
+
+    function get() public view returns (uint) {
+        return storedData;
+    }
+}
+```
+
+##### Smart-Contract Storage Example in *Java*:
+```java
+public class SimpleStorage {
+    private Uint256 storedData;
+
+    public void setStoredData (Uint256 storedData) {
+        this.storedData = storedData;
+    }
+
+    public Uint256 getStoredData () {
+        return storedData;
+    }
+}
+```
+
+## Rationale
+![](https://imgur.com/hs8WlUw.jpg)
+
+### Minting
+Overall scope/goal for this proposal is for these tokens to be minted as easy as NFTs. This could be accomplished simply from a "button" within a dApp/App that mints a new token, *other simpler possibilities open to discussion*. The Total & Circulating Supply of these tokens are to be minted by dApps/Apps integrated within the logic-base implemented by the developer.
+
+### Exchanging
+These New-ERC Tokens are meant to act as ***"Personal Currencies"*** so in order for them to be successfully exchanged, they will need the consent of both parties *(like trading one NFT for another OR specifically speaking, trading 1 Yuan for 1 Dollar, or 50 Yuans for 1 Dollar, etc)*.
```

Comment:
The rationale section should explain choices made in the specification section, not introduce new specifications.

Reply:
Understood.